presto-docs/src/main/sphinx/sql/create-role.rst

```diff
@@ -0,0 +1,51 @@
+===========
+CREATE ROLE
+===========
+
+Synopsis
+--------
+
+.. code-block:: none
+
+    CREATE ROLE role_name
+    [ WITH ADMIN ( user | USER user | ROLE role | CURRENT_USER | CURRENT_ROLE ) ]
+    [ IN catalog ]
+
+Description
+-----------
+
+``CREATE ROLE`` creates the specified role in ``catalog`` or in the
+current catalog if ``catalog`` is not specified.
+
+The optional ``WITH ADMIN`` clause causes the role to be created with
+the specified user as a role admin. If the optional ``WITH ADMIN`` clause is not
+specified, the role is created with current user as an admin.
+
+Examples
+--------
+
+Create role ``admin`` ::
+
+    CREATE ROLE admin;
+
+Create role ``moderator`` with admin ``admin``::
+
+    CREATE ROLE moderator WITH ADMIN USER admin;
```

Comment:
having "USER admin" here and creating role admin above is confusing -- can you either have ROLE admin or USER bob?